src/physics/utils/cam_constituents.F90

```diff
@@ -312,10 +316,14 @@ subroutine const_get_index(name, cindex, abort, warning, caller)
       character(len=*), parameter :: subname = 'const_get_index: '
       !-----------------------------------------------------------------------
 
-      ! Find tracer name in the master table
-      call cam_const_get_index(name, cindex, errcode=errcode, errmsg=errmsg)
+      call ccpp_const_get_idx(const_props, name, cindex, errmsg, errcode)
 
       if (errcode /= 0) then
+         call endrun(subname//"Error "//to_str(errcode)//": "//           &
```

Comment:
I might use `stringify` here instead of `to_str` (my hope is to eventually use `stringify` for all string conversions):

```suggestion
         use string_utils, only: stringify
         ...
         call endrun(subname//"Error "//stringify((/errcode/))//": "//           &
```

Reply:
Thanks, updated!